src/components/testCaseConfiguration/testCaseData/TestCaseData.tsx

```diff
@@ -68,7 +70,28 @@ const TestCaseData = () => {
     setToastOpen(open);
   };
 
-  const handleSubmit = async (values) => {};
+  const handleSubmit = async (values) => {
+    testCaseService.current
+      .shiftAllTestCaseDates(
+        measure.id,
+        parseInt(formik.values.shiftTestCaseDates)
+      )
+      .then(() => {
+        handleToast(
+          "success",
+          `Test Case Shift Dates for measure: ${measure.id} successful.`,
+          true
+        );
+      })
+      .catch((err) => {
+        handleToast(
+          "danger",
+          `Unable to shift test Case dates for measure: ${measure.id}. Please try again. If the issue continues, please contact helpdesk.`,
```

Comment:
I think it should be `Test Case dates could not be shifted. Please try again` as per ticket unless it is changed later.

Reply:
@adongare Thanks Ashok, just changed the message.